target/linux/ramips/dts/ATEL-U270.dts

```diff
@@ -0,0 +1,177 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+/ {
+	compatible = "ATEL-U270", "ralink,mt7620a-soc";
+	model = "ATEL U270";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+		wan {
+			label = "atel-u270:orange:wan";
+			gpios = <&gpio2 27 1>;
+		};
+		wifi {
+			label = "atel-u270:orange:wifi";
+			gpios = <&gpio3 0 1>;
+		};
+		wps {
+			label = "atel-u270:orange:wps";
+			gpios = <&gpio2 28 1>;
+		};
+		power {
+			label = "atel-u270:orange:power";
+			gpios = <&gpio2 20 1>;
+		};
+		power-red {
+			label = "atel-u270:red:power";
+			gpios = <&gpio2 21 1>;
+		};
+  signal1 {
+   label = "atel-u270:white:signal1";
+   gpios = <&gpio2 26 1>;
+  };
+  signal2 {
+   label = "atel-u270:white:signal2";
+   gpios = <&gpio2 23 1>;
+  };
+  signal3 {
+   label = "atel-u270:white:signal3";
+   gpios = <&gpio2 24 1>;
+  };
+  signal4 {
+   label = "atel-u270:white:signal4";
+   gpios = <&gpio2 25 1>;
+  };
+  signal5 {
+   label = "atel-u270:white:signal5";
+   gpios = <&gpio2 22 1>;
+  };
+  signal-blue {
+   label = "atel-u270:blue:signal";
+   gpios = <&gpio1 7 1>;
+  };
+  signal-red {
+   label = "atel-u270:red:signal";
+   gpios = <&gpio1 8 1>;
+  };
+  signal-green {
+   label = "atel-u270:green:signal";
+   gpios = <&gpio1 9 1>;
+  };
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+/*		power {
```

Comment:
What is wrong with the keys? Either fix it or drop the comment. We do not keep not working stuff as comment.